### DISCUSSION TITLE
Customization of ai-data-analysis-MulitAgent

### DISCUSSION THREAD

**Discussion by nakulz** [starpig1129/Multi-agent-DataAnalysis#1] [General]
Seeking clarification on two items:
	1.	In #2 of the “Usage” section of the documentation, what is meant by “Sata data.csv in data_storage”? Is “Sata” a typo, or does it refer to something specific?
	2.	For the customization step in the main.ipynb file, where can I find the default OnlineSalesData.csv file, and what are its contents?

**Comment by zi-yue-1129** [comment]
Thank you for your questions about my project. As the author, I'd like to
address your concerns:

1. Thank you for pointing out this issue. You are correct; there is indeed
a typographical error. "Sata data.csv" should be "Set data.csv". The
intention of this step is to instruct users to place the data file they
want to analyze in the data_storage folder. I will correct this error in
the documentation.

2. Regarding the OnlineSalesData.csv file, this was a sample data file I
used during development for testing purposes. I didn't include it in the
project as it was only for my personal testing. If you need example data to
start using this project, I suggest you could use your own  data

nakulz ***@***.***> 於 2024年9月9日 週一 21:14 寫道：

**Reply by nakulz** [reply]
Thank you for your quick response and for making update already. 

Will be great if you can add a sample / dummy YourDataName.csv with some explanation on what it needs to contain. 

Eager to give this project a go.

**Comment by zi-yue-1129** [comment]
Because there is actually no certain data standard
As long as it is text data
But I hope to support different types such as images in the future



nakulz ***@***.***> 於 2024年9月9日 週一 下午10:35寫道：

**Comment by Divyanshu6496** [comment]
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[8], line 1
----> 1 print_stream(events)

NameError: name 'events' is not defined

**Reply by zi-yue-1129** [reply]
Are you sure you have the complete execution of the program?
Because 'events' are just two cells above

**Reply by Divyanshu6496** [reply]
So here is how I am following.

Download >> conda
Got keys from OpenAI 
In Jupyter notebook >> followed all commands 
put my data on it.

correct me where I am wrong

**Reply by zi-yue-1129** [reply]
![image](https://github.com/user-attachments/assets/6853cfde-457c-40d0-963d-4eebd63fca77)
But if there are correct execution all programs should not appear
'events' is not defined
You can see that the first cell in the picture has a definition
Have you made any changes to the code?
